src/main/java/core/mate/academy/service/impl/ExcavatorProducer.java

```diff
@@ -0,0 +1,13 @@
+package core.mate.academy.service.impl;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.service.MachineProducer;
+
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
+        return List.of(new Excavator());

```

Comment:
The `get` method should return a list containing 2-3 `Excavator` instances as per the task requirements. Consider creating additional `Excavator` objects and adding them to the list.